tests/utils/summary_category_extractor.py

```diff
@@ -0,0 +1,48 @@
+import langchain
+from langchain.cache import SQLiteCache
+from langchain.chat_models import ChatOpenAI
+from langchain.prompts import (
+    ChatPromptTemplate,
+    SystemMessagePromptTemplate,
+    HumanMessagePromptTemplate,
+)
+
+
+def summary_category_extractor(summary: str, question_1: str, question_2: str):
```

Comment:
rename to assert_summary_using_qa (summary, questions)
and internally perform the assertions.